express-get-json/index.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+const arr = [];
```

Comment:
It's best to keep variables scoped as narrowly as possible. Instead of declaring this in global scope, you should declare it down where it's used.